test/check-application

```diff
@@ -1517,6 +1561,37 @@ class TestFiles(testlib.MachineCase):
         b.key("ArrowLeft")
         b.wait_visible("[data-item='file0'].row-selected")
 
+        # Go up and down in directory hierarchy
+        b.click("[data-item='testdir']")
+        b.key("ArrowDown", modifiers=["Alt"])
+        self.assert_last_breadcrumb("testdir")
+        b.key("ArrowUp", modifiers=["Alt"])
+        b.wait_visible("[data-item='testdir']")
+
+        # Manually edit path
+        b.key("L", modifiers=["Control"])
+        b.input_text("/home/admin/anotherdir")
+        b.key("Enter")
+        self.assert_last_breadcrumb("anotherdir")
+
+        # Go back in history twice
+        # HACK: browsers don't understand alt + left/right when using b.key()
+        # b.key("ArrowLeft", repeat=2, modifiers=["Alt"])
```

Comment:
This tries to test the native hotkeys of the browser which we can't do with BiDI, this is also we have `b.reload()`